geopandas/io/tests/test_io.py

```diff
@@ -17,14 +17,26 @@ def setUp(self):
             self.columns = list(f.meta["schema"]["properties"].keys())
 
     def test_to_postgis(self):
-        import sqlalchemy
+        try:
+            import sqlalchemy
+        except ImportError:
+            raise unittest.case.SkipTest()
+
         engine = sqlalchemy.create_engine("postgres://localhost/test_geopandas")
         try:
-            engine.connect()
+            con = engine.connect()
         except sqlalchemy.exc.OperationalError:
             raise unittest.case.SkipTest()
 
-        self.df.to_postgis("nybb_write", engine, if_exists="replace")
+        try:
+            db_name = "nybb_write"
+            self.df.to_postgis(db_name, engine, if_exists="replace")
+            back = read_postgis("SELECT * FROM {};".format(db_name), engine,
+                                geom_col="geometry")
+        finally:
+            con.close()
+        back = back.rename(columns=lambda x: x.lower())
```

Comment:
I had to add the same sort of handling for case in #379. I guess I should revisit the suggestion to rework `validate_boro_df`.